lightwood/encoder/categorical/autoencoder.py

```diff
@@ -24,18 +22,15 @@ def __init__(self, stop_after: int = 3600, is_target: bool = False, max_encoded_
         self.decoder = None
         self.onehot_encoder = OneHotEncoder(is_target=self.is_target)
         self.desired_error = 0.01
-        self.use_autoencoder = use_autoencoder
         self.stop_after = stop_after
         # @TODO stop using instead of ONEHOT !!!@!
         self.is_nn_encoder = True
         self.output_size = None
-        if self.is_target:
-            self.max_encoded_length = None
-        else:
-            self.max_encoded_length = max_encoded_length
+        self.max_encoded_length = max_encoded_length
 
     def _train_callback(self, error, real_buff, predicted_buff):
-        log.info(f'{self.name} reached a loss of {error} while training !')
+        pass
+        #  log.info(f'{self.name} reached a loss of {error} while training !')
```

Comment:
Leftover comment, though I think it may be useful to leave this one as a `log.debug()` call

Reply:
whole function can be removed tbh, a debug log would be useful but I'll leave it w/o to motivate us to phase out the Gym when we get to resisting this encoder.